Centaurus/Program.cs

```diff
@@ -0,0 +1,53 @@
+using System;
+using System.Diagnostics;
+using System.IO;
+using CommandLine;
+using NLog;
+
+namespace Centaurus
+{
+    public class Program
+    {
+
+        private static Logger logger = LogManager.GetCurrentClassLogger();
+
+        public static void Main(string[] args)
+        {
+            var mergedArgs = CommandLineHelper.GetMergedArgs<AuditorSettings>(CommandLineHelper.GetMergedArgs<AlphaSettings>(args));
+
+            Parser.Default.ParseArguments<AlphaSettings, AuditorSettings>(mergedArgs)
+                .WithParsed<AlphaSettings>(s => ConfigureAndRun(s))
+                .WithParsed<AuditorSettings>(s => ConfigureAndRun(s))
+                .WithNotParsed(e =>
+                {
+                    Console.WriteLine("Press Enter to quit");
+                    Console.ReadLine();
+                });
+        }
+
+        static void ConfigureAndRun<T>(T settings)
+            where T : BaseSettings
+        {
+            var isAlpha = settings is AlphaSettings;
+            Console.Title = isAlpha ? "Alpha" : $"Auditor";
+
+            settings.Build();
+
+            var logsDirectory = Path.Combine(settings.CWD, "logs");
```

Comment:
Do you think that we should introduce a separate config option for the logs destination? CWD directory might be protected from write (unlikely, but still).

Reply:
We need a directory to place the log files there. I believe that we need to document this and clarify what permission we need.